Survey/Sources/Data/Repositories/SurveyRepository.swift

```diff
@@ -43,4 +44,18 @@ final class SurveyRepository: SurveyRepositoryProtocol {
             .map { $0 as APISurvey }
             .eraseToAnyPublisher()
     }
+
+    func submit(id: String, questionSubmissions: [QuestionSubmission]) -> Observable<Bool> {
+        networkAPI
+            .performRequest(
+                SurveyRequestConfiguration.submit(
+                    id: id,
+                    questionSubmissions: questionSubmissions.map {
+                        APIQuestionSubmission(questionSubmission: $0)
+                    }
+                )
+            )
+            .map { $0 as Bool }
```

Comment:
Could you check if we need this line? PerformRequest should return T right 🤔

Reply:
Nice catch 🙌 . It's a redundancy code.